### PR TITLE
Add QFN-16_1EP_3x3mm_P0.5mm_EP1.6x1.6mm

### PR DESCRIPTION
**Pull request by whitequark**
See https://github.com/KiCad/kicad-footprints/pull/2049#issuecomment-573465158.

### CODE REVIEW

**Comment by chschlue** [comment]
Pending #422

**Comment by whitequark** [comment]
@chschlue #422 is merged--is there anything I need to do here?

**Comment by chschlue** [comment]
You need to regenerate https://github.com/KiCad/kicad-footprints/pull/2049 using the current scripts (i.e. past-422.)
Also, this PR has to be reviewed before 2049 can be merged (meaning 2049 can wait until your size definitions here have been verified so you don't have to update the FP PR twice.)

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml [hunk 1]
```diff
@@ -170,6 +170,39 @@ QFN-16-1EP_3x3mm_P0.5mm_EP1.45x1.45mm:
...
+
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x: 1.6
+  EP_size_y: 1.6
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml]: Use package pad dimensions and tolerances as given in the datasheet and then add EP_size_[xy]_overwrite to get a 1.6x1.6mm FP pad.

**Comment by whitequark** [comment]
Thanks for the review, I've applied all changes.

**Comment by chschlue** [comment]
Looks good now. Please push the new FP to https://github.com/KiCad/kicad-footprints/pull/2049.

**Comment by whitequark** [comment]
Done.

**Comment by chschlue** [comment]
FP merged.

**Pull request merged**